scripts/generate_test_suites.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+parts = sys.argv[1]
+
+instrumentation_tests_path = 'instrumentation-tests/src/androidTest/java/com/mapbox/navigation/instrumentation_tests'
+core_tests = os.listdir(f'{instrumentation_tests_path}/core')
+ui_tests = os.listdir(f'{instrumentation_tests_path}/ui')
+
+core_part_size = int(len(core_tests) / int(parts)) + 1
```

Comment:
NIT: what about UI? Does the passed argument mean "total number of parts" or "number of core parts" and total number will be this argument + 1?

Reply:
means "number of core parts", I need to clarify it